introduction.md

```diff
@@ -1,3 +1,8 @@
 Hi, my name is David Young!
 
 My favorite Code School path so far is Ruby.
+
+Deadly skills:
+*Ruby
+*JavaScript
+*Python
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?